File: Modul_3/Soal_1/PRAK301-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    int angka1;
    int angka2;

    scanf("%d %d", &angka1, &angka2);
    if(angka1 > angka2) {
        printf("%d %d", angka2, angka1);
    }
    else{
        printf("%d %d", angka1, angka2);
    }
}

File: Modul_3/Soal_1/PRAK301-2310817210015-MuhammadBukhariFitri.py
angka1 = input().split()

if len(angka1) == 2:
    angka1, angka2 = map(int, angka1)

else:
    angka1 = int(angka1[0])
    angka2 = int(input())

if (angka1 > angka2):
    print(f"{angka2} {angka1}")
else:
    print(f"{angka1} {angka2}")
